array.c

```diff
@@ -6856,6 +6946,40 @@ rb_ary_deconstruct(VALUE ary)
  *     arr.keep_if {|a| a < 4}   #=> [1, 2, 3]
  *     arr                       #=> [1, 2, 3]
  *
+ *  == Chaining \Method Calls
```

Comment:
I think this section should move to the [method syntax](https://github.com/ruby/ruby/blob/master/doc/syntax/calling_methods.rdoc) document and be linked-to from here, as it is general behavior that is not specific to Array.

Reply:
Done.